src/pages/CountryInfo.js

```diff
@@ -37,41 +33,51 @@ const CountryInfo = () => {
               <div className="details">
                 <div className="info-left">
                   <p>
-                    Offcial Name: <span>{countryInfo[0].name.official}</span>
+                    Offcial Name:
+                    {' '}
+                    <span>{countryInfo[0].name.official}</span>
                   </p>
                   <p>
-                    Population: <span>{countryInfo[0].population}</span>
+                    Population:
+                    {' '}
+                    <span>{countryInfo[0].population}</span>
                   </p>
                   <p>
-                    Contenint: <span>{countryInfo[0].continents}</span>
+                    Contenint:
+                    {' '}
+                    <span>{countryInfo[0].continents}</span>
                   </p>
 
                   <p>
-                    Time Zones: <span>{countryInfo[0].timezones[0]}</span>
+                    Time Zones:
+                    {' '}
+                    <span>{countryInfo[0].timezones[0]}</span>
                   </p>
                   <p>
-                    Capital: <span>{countryInfo[0].capital}</span>
+                    Capital:
+                    {' '}
+                    <span>{countryInfo[0].capital}</span>
                   </p>
                 </div>
 
                 <div className="info-right">
                   <p>
-                    Currencies: <span>
+                    Currencies:
+                    {' '}
+                    <span>
                       {Object.values(countryInfo[0].currencies)
-                        .map((item) => {
-                          return item.name;
-                        })
-                        .join(", ")}
+                        .map((item) => item.name)
+                        .join(', ')}
                     </span>
                   </p>
 
                   <p>
-                    Languages:  <span>
+                    Languages:
+                    {' '}
+                    <span>
                       {Object.values(countryInfo[0].languages)
-                        .map((item) => {
-                          return item;
-                        })
-                        .join(", ")}
+                        .map((item) => item)
+                        .join(', ')}
                     </span>
```

Comment:
- [ ] It would be better if we do not display the image on the details page. When users click on a country, they are taken to a details page that should have more info about a country, unfortunately, I only saw an image there. Kindly fix this issue.